packages/components/src/summary-button/style.scss

```diff
@@ -41,6 +41,18 @@
 
 	.summary-button-title {
 		@include body-large;
+
+		&.has-text-align-left {
+			text-align: start;
+		}
```

Comment:
Franky I would just slap `text-align: start` on the component itself without adding a prop. I can't see a scenario where text-align: right is needed.